app/services/check_answers/applicant_section.rb

```diff
@@ -3,18 +3,25 @@ class ApplicantSection
     include ActiveModel::Model
     include ActiveModel::Attributes
 
-    FIELDS = %i[over_60
+    FIELDS = %i[domestic_abuse
```

Comment:
not sure domestic_abuse should be a field - it's derived from proceeding type?

Reply:
You're right, I forgot that we have a mechanism in check answers to handle enums like `proceeding_type` so I didn't need to convert it to a boolean.